critiquebrainz/db/review.py

```diff
@@ -316,33 +319,9 @@ def create(*, entity_id, entity_type, user_id, is_draft, text=None, rating=None,
 
 
 # pylint: disable=too-many-branches
-def list_reviews(*, inc_drafts=False, inc_hidden=False, entity_id=None,
-                 entity_type=None, license_id=None, user_id=None,
-                 language=None, exclude=None, sort=None, limit=20,
-                 offset=None):
-    """Get a list of reviews.
-
-    This function provides several filters that can be used to select a subset of reviews.
-
-    Args:
-        entity_id (uuid): ID of the entity that has been reviewed.
-        entity_type (str): Type of the entity that has been reviewed.
-        user_id (uuid): ID of the author.
-        sort (str): Order of the returned reviews. Can be either "popularity" (order by difference in +/- votes),
-                    or "published_on" (order by publish time), or "random" (order randomly).
-        limit (int): Maximum number of reviews to return.
-        offset (int): Offset that can be used in conjunction with the limit.
-        language (str): Language code of reviews.
-        license_id (str): License ID that reviews are associated with.
-        inc_drafts (bool): True if reviews marked as drafts should be included, False if not.
-        inc_hidden (bool): True if reviews marked as hidden should be included, False if not.
-        exclude (list): List of reviews (their IDs) to exclude from results.
-
-    Returns:
-        Tuple with two values:
-        1. list of reviews as dictionaries,
-        2. total number of reviews that match the specified filters.
-    """
+def get_reviews_list(connection, *, inc_drafts=False, inc_hidden=False, entity_id=None,
```

Comment:
you should add documentation here for why we're passing the `connection` here.

Reply:
@paramsingh Done!